cpp/src/arrow/engine/substrait/extension_set.cc

```diff
@@ -979,23 +981,22 @@ ExtensionIdRegistry::SubstraitAggregateToArrow DecodeBasicAggregate(
             options = std::make_shared<compute::VarianceOptions>(ddof);
           }
         }
-        fixed_arrow_func += arrow_function_name;
 
-        ARROW_ASSIGN_OR_RAISE(compute::Expression arg, call.GetValueArg(0));
-        const FieldRef* arg_ref = arg.field_ref();
-        if (!arg_ref) {
-          return Status::Invalid("Expected an aggregate call ", call.id().uri, "#",
-                                 call.id().name, " to have a direct reference");
+        fixed_arrow_func += arrow_function_name;
```

Comment:
This line was glued to the previous block before it's part of the function name building process.

Reply:
Sorry I am not sure what do you mean here - do you want me to change anything?

Reply:
To keep the line glued to the previous block as it was before. It's a nitpick, but makes a potential function extraction in the future more obvious.

Reply:
I see. Updated